plexpy/notification_handler.py

```diff
@@ -516,11 +563,35 @@ def build_notify_text(session=None, timeline=None, state=None):
                         'summary': metadata['summary'],
                         'tagline': metadata['tagline'],
                         'rating': metadata['rating'],
-                        'duration': duration
+                        'duration': duration,
+                        'action': state,
+                        'streams': stream_count
                         }
 
     # Default subject text
     subject_text = 'PlexPy (%s)' % server_name
+    # Default scripts args
+    script_args = ''
```

Comment:
Does this work instead?
```
# Default scripts args
script_args = ''

# Add to line 406 and 421
script_args_text = strip_tag(re.sub(pattern, '', plexpy.CONFIG.NOTIFY_SCRIPTS_ARGS_TEXT))

if script_args_text:
    try:
        script_args = [unicode(arg).format(**available_params) for arg in script_args_text .split()]
    except LookupError, e:
        logger.error(u"PlexPy Notifier :: Unable to parse field %s in script argument. Using fallback." % e)
    except:
        logger.error(u"PlexPy Notifier :: Unable to parse custom notification subject. Using fallback.")
```
And don't need `quoute_stuff`. (Spelled "quote" by the way)